src/window/city.c

```diff
@@ -92,11 +92,23 @@ static void draw_time_left(void)
     }
 }
 
+static void draw_speedrun_infos(void)
+{
+    if (config_get(CONFIG_SCREEN_DISPLAY_SPEEDRUN_INFOS)) {
+        int x, y, width, height;
+        city_view_get_viewport(&x, &y, &width, &height);
```

Comment:
`x` is guaranteed to be always 0, and you can use `screen_height()` to get the height.